parts/kubernetesagentcustomdata.yml

```diff
@@ -66,7 +66,7 @@ write_files:
     - name: localcluster
       cluster:
         certificate-authority: /etc/kubernetes/certs/ca.crt
-        server: https://{{WrapAsVariable "masterInternalLbIp"}}:443
+        server: https://{{WrapAsVariable "masterIP"}}:443
```

Comment:
The way this PR handle this, is to replace everywhere using masterIP. 
To me, it is a little bit confusing, when i saw "masterIP", which actually is the "masterInternalLbIp". 

Wondering, is there a way to make the name more explicit?